### ISSUE TITLE
French text is not read properly (Android)

### ISSUE THREAD

**Issue by sarasuri** [cds-snc/covid-shield-mobile#256]
Scenario

Samsung Galaxy J3 Prime 
Android version 7.0

With Talkback on, on the "Join the effort to slow the spread" screen, text “Français" is read as English text with English accent/pronunciation.

Expected result
“Français" should be read as French text.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts

Screenshots
![french](https://user-images.githubusercontent.com/12256067/85948875-3efc4180-b921-11ea-980c-4dd2dca24bc3.gif)

**Comment by JuliannaR** [comment]
@katewilhelm @timarney can you confirm this has been removed?

**Comment by katewilhelm** [comment]
I know the intention is to remove it but not sure where actually removing it falls in priorities

**Comment by JuliannaR** [comment]
This has been removed and is no longer an option in the flow.
